apps/src/applab/appStorage.js

```diff
@@ -38,6 +38,10 @@ var handleGetKeyValue = function(onSuccess, onError) {
   if (this.readyState !== 4) {
```

Comment:
The XHR documentation? Once the javascript pipeline is unified *and* our test framework supports jQuery, this API will get combined with the (ugh) 2 other duplicate client libaries for the same server API. That said I could extract a constant here if it would be helpful.